widgetschema.json

```diff
@@ -228,8 +262,54 @@
                             "items": { "$ref": "#/definitions/dataType" }
                         }
                     }
+                },
+                "visibility": { "$ref": "#/definitions/widgetVisibilityGroup" },
+                "fields": {
+                    "type": "array",
+                    "items": { "$ref": "#/definitions/widgetProperty" },
+                    "description": "Fields for the multiple property"
+                },
+                "itemDisplayName": {
+                    "type": "string",
+                    "description": "Specifies the label for the singular repeated item within a multiple property"
                 }
             }
+        },
+        "widgetVisibilityGroup": {
+            "type": "object",
+            "description": "Group of conditions for the visibility of a widget property",
+            "properties": {
+                "operator": {
+                    "type": "string",
+                    "enum": ["and", "or"]
+                },
+                "conditions": {
+                    "type": "array",
+                    "items": { "$ref": "#/definitions/widgetVisibilityCondition" }
+                }
+            },
+            "required": ["operator", "conditions"]
+        },
+        "widgetVisibilityCondition": {
+            "type": "object",
+            "description": "Condition for the visibility of a widget property",
+            "properties": {
+                "field": { "type": "string" },
```

Comment:
This is a reference to a widget property, similar to the values in `mappedValues`?